includes/controllers/class.llms.controller.certificates.php

```diff
@@ -117,10 +117,33 @@ public function maybe_handle_reporting_actions() {
 			$this->download( $cert_id );
 		} elseif ( isset( $_POST['llms_delete_cert'] ) ) {
 			$this->delete( $cert_id );
+		} elseif ( isset( $_POST['llms_enable_cert_sharing'] ) || isset( $_POST['llms_disable_cert_sharing'] ) ) {
+			$this->change_sharing_settings( $cert_id, isset( $_POST['llms_enable_cert_sharing'] ) );
 		}
 
 	}
 
+	/**
+	 * Change shareable settings of a certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int  $cert_id WP Post ID of the llms_my_certificate
+	 * @param bool $is_allowed Allow share the certificate or not
```

Comment:
```suggestion
	 * @param int  $cert_id WP Post ID of the llms_my_certificate.
	 * @param bool $is_allowed Allow share the certificate or not.
```
Please bear with us, this will break a rule that is currently disabled:
https://github.com/gocodebox/lifterlms/blob/b681ffbd6b9b36450700383917d0d150982170a8/phpcs.xml#L30
We are slowly fixing these (tracked in #946)